source/index.styl

```diff
@@ -1,38 +1,35 @@
-.likely,
-.likely__widget {
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  display: inline-block;
-  margin: 0;
-  padding: 0;
-  text-indent: 0 !important;
-  list-style: none !important;
-  font-weight: normal;
-  font-family: Helvetica Neue, Arial, sans-serif;
-  font-size: inherit;
-}
-
 // Hide buttons until they are loaded
 .likely {
   opacity: 0;
-  font-size: 0 !important;
-}
-.likely_visible {
-  opacity: 1;
   transition: opacity .1s ease-in;
-}
-.likely > * {
-  display: inline-block;
-  visibility: hidden;
-}
-.likely_visible > * {
-  visibility: inherit;
+
+  * {
+    visibility: hidden;
+  }
+
+  &_visible {
+    opacity: 1;
+
+    * {
+      visibility: inherit;
+    }
+  }
 }
 
 .likely {
-  // outline: 1px #f00 solid;
-  &__widget {
```

Comment:
Why don't we need the same for &__widget anymore?

Reply:
As for this particular block, `display: inline-block` part was removed as duplicate (it was already covered by `.likely > *` selector, and the other two (`position: relative;`, `white-space: nowrap;`) are still there, moved below in the code (line 32).

In general, I also tried not to duplicate rules that are going to be inherited from .likely parent, since we currently don't support having a likely_widget on its own (it's always wrapped in .likely).

Reply:
Gotcha, thanks!